lib/internal/crypto/random.js

```diff
@@ -314,81 +340,27 @@ function getBufferedUUID() {
 
   if (uuidBatch === 0) randomFillSync(uuidData);
   uuidBatch = (uuidBatch + 1) % kBatchSize;
-  return uuidData.slice(uuidBatch * 16, (uuidBatch * 16) + 16);
+  return serializeUUID(uuidData, uuidBatch * 16);
+}
+
+function getUnbufferedUUID() {
+  let uuidBuf = uuidNotBuffered;
+  if (uuidBuf === undefined)
+    uuidBuf = uuidNotBuffered = secureBuffer(16);
```

Comment:
```suggestion
  const uuidBuf = uuidNotBuffered ||= secureBuffer(16);
  let uuidBuf = uuidNotBuffered;
  if (uuidBuf === undefined)
    uuidBuf = uuidNotBuffered = secureBuffer(16);
```

Reply:
I used nullish assignment without a local variable, it keeps a similar performance while it simplifies the code too